### PR TITLE
Add support of dashboard timezone in Start and End time

### LINKED ISSUE #120: Wrong Start Time and End time
Hi,

We have webbased calendar connected to MySQL Database, i made a query to show in the Grafana Calendar it was successfully although i noticed that the start time and end time were wrong, I also changed the "Change time settings" but there was no change.
wrong

the start time and end time should be 11:00AM to 12:00PM

here is my query 
SELECT title,start,end, location, creation_date FROM `calendar_events` ORDER BY creation_date

thanks in advanced

### PR DESCRIPTION
**Pull request by asimonok**
Resolves #120

### CODE REVIEW

**Review by mikhail-vl — CHANGES_REQUESTED** [review]
As discussed, please fix the performance degradation of `useCalendarEvents`

File: src/utils/calendarEvents.ts [hunk 1]
```diff
@@ -85,26 +86,74 @@ export const useColors = (fieldConfig?: FieldConfigSource) => {
...
+  /**
+   * Browser Date
+   * Reset milliseconds to prevent losing 1 minute in difference
+   */
+  const browserDate = dayjs(date).set('milliseconds', 0);
+
+  /**
+   * Time Zone Date
+   */
+  const timeZoneDate = dayjs(date.toLocaleString('en-US', { timeZone }));
```
Comment (mikhail-vl) [inline src/utils/calendarEvents.ts:122]: It should be set to a selected locale.

File: src/utils/calendarEvents.ts [hunk 2]
```diff
@@ -133,13 +182,13 @@ export const useCalendarEvents = (
             text,
             description,
             labels,
-            start: dayjs(start),
+            start: dayjs(start).minute(minutesOffset),
```
Comment (mikhail-vl) [inline src/utils/calendarEvents.ts:185]: We are losing minutes because it's set to 0.

**Review by mikhail-vl — CHANGES_REQUESTED** [review]

File: src/utils/calendarEvents.ts (hunk 1 above)
Comment (asimonok) [reply src/utils/calendarEvents.ts:122]: the date is being used only for timezone offset calculation so we can use any locale here

File: src/utils/calendarEvents.ts (hunk 2 above)
Comment (asimonok) [reply src/utils/calendarEvents.ts]: fixed

**Review by mikhail-vl — APPROVED** [review]
Looks great!

**Pull request merged**